newsletter-march-2023.rst

```diff
@@ -0,0 +1,126 @@
+  If you want to see the change for your project,
+  have a look at your build times before and after **February 8**.
+- 1️⃣️ We are going to have 1 main Read the Docs website in the future.
+  The first move has been done now,
+  as readthedocs.com now points to `about.readthedocs.com <https://about.readthedocs.com>`__ instead of being a separate site for Read the Docs for Business.
+  The separate landing page on readthedocs.org will also be moving.
+- 📚️ Documentation refactor underway:
+  A regular reader of our :doc:`user docs <readthedocs:index>` might notice that our navigation has been greatly changed and now contains only the `4 Diátaxis categories <https://diataxis.fr/>`__.
+- 🧹️ We are now able to delist projects from search engines more efficiently.
+  If you find an unofficial and unmaintained project,
+  :doc:`please see our policy on unofficial and unmaintained projects <readthedocs:unofficial-projects>`.
+- 🐞 Custom 404s with Sphinx DirHTML builder `are now supported <https://github.com/readthedocs/sphinx-notfound-page/issues/215>`__.
+- 🐞 :doc:`Pull Request <readthedocs:guides/pull-requests>` builds now link directly to the preview version of the documentation changes.
+- 🔒️ Vulnerability fixed: Pull requests preview builds `GHSA-h4cf-8gv8-4chf <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-h4cf-8gv8-4chf>`__
+- 🔒️ Vulnerability fixed: Access to files from any project possible with crafted path traversal, see: `GHSA-5w8m-r7jm-mhp9 <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-5w8m-r7jm-mhp9>`__.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. _rclone: https://rclone.org/
+
+
+Upcoming features
+-----------------
+
+..
+  Notes:
+
+  Next newsletter:
+  Make a general announcement of our Roadmap
+
+  General:
+
+  When creating newsletter drafts, we keep the items here from the previous newsletter.
+  This is in order to ensure due follow-up on features that are announced publicly.
+  
+  Feature done? A great follow-up is to add what was previously an upcoming feature as a released feature in the former section.
+  
+  Feature not done?
+  Make sure that upcoming features are announced with a link to issues or PRs where the progress can be seen.
+  If this is done, then subsequent newsletters aren't compelled to share progress when it's uninteresting.
+  
+  If a feature was announced as upcoming but isn't yet released,
+  then try rephrasing the announcement as a general news update about the progress and where it can be followed.
+
+- 🪄️ The Django application that handles uncached requests on all our hosted documentation is called El Proxito.
+  And that application is getting a major refactor,
+  improving both speed and features.
+  Most of the refactoring work is already merged and within the coming weeks,
+  we will start to roll it out as a kind of A/B testing.
```

Comment:
```suggestion
  we will start to roll it out gradually.
```

This should be better... A/B testing isn't it.

Reply:
Yea, this is better 👍